Backend/Remora.Discord.API.Abstractions/API/Objects/Interactions/Components/SelectMenu/ISelectMenuComponent.cs

```diff
@@ -74,7 +79,12 @@ public interface ISelectMenuComponent : IMessageComponent, IPartialSelectMenuCom
     Optional<string> IPartialSelectMenuComponent.CustomID => this.CustomID;
 
     /// <inheritdoc/>
-    Optional<IReadOnlyList<IPartialSelectOption>> IPartialSelectMenuComponent.Options => new(this.Options);
+    Optional<IReadOnlyList<IPartialSelectOption>> IPartialSelectMenuComponent.Options => this.Options.HasValue
```

Comment:
is `new(this.Options)` or `this.Options` not sufficient here via co/contravariance?

Reply:
It doesn't appear so. I'd assumed the same but it won't compile.